File: CS/CustomPrintPreviewTest/CustomGridViewPrintingTest.cs
using System.Drawing;
using NUnit.Framework;
using System.Reflection;
using DevExpress.XtraGrid;
using CustomPrintPreview.Grid;
using DevExpress.XtraGrid.Columns;
using DevExpress.XtraGrid.Views.Base;

namespace CustomPrintPreviewTest {
    [TestFixture]
    public class CustomGridViewPrintingTest {
        [Test]
        public void InitViewStylesTest() {
            GridControl gc = new CustomGridControl();
            gc.ForceInitialize();
            gc.MainView = new CustomGridView(gc);
            GridColumn col = ((ColumnView)gc.MainView).Columns.AddField("Column");
            using (CustomGridViewPrinting cgvp = new CustomGridViewPrinting(gc, col)) {
                cgvp.InitFrame(gc.MainView, "GridView", new Bitmap(16, 16));
            typeof(CustomGridViewPrinting).InvokeMember("InitViewStyles", BindingFlags.Instance |
                BindingFlags.NonPublic | BindingFlags.InvokeMethod, null, cgvp, new object[0]);
            }
        }
    }
}

File: VB/CustomPrintPreviewTest/CustomGridViewPrintingTest.vb
Imports System.Drawing
Imports NUnit.Framework
Imports System.Reflection
Imports DevExpress.XtraGrid
Imports CustomPrintPreview.Grid
Imports DevExpress.XtraGrid.Columns
Imports DevExpress.XtraGrid.Views.Base

Namespace CustomPrintPreviewTest
    <TestFixture> _
    Public Class CustomGridViewPrintingTest
        <Test> _
        Public Sub InitViewStylesTest()
            Dim gc As New CustomGridControl()
            gc.ForceInitialize()
            gc.MainView = New CustomGridView(gc)
            Dim col As GridColumn = CType(gc.MainView, ColumnView).Columns.AddField("Column")
            Using cgvp As New CustomGridViewPrinting(gc, col)
                cgvp.InitFrame(gc.MainView, "GridView", New Bitmap(16, 16))
            GetType(CustomGridViewPrinting).InvokeMember("InitViewStyles", BindingFlags.Instance Or BindingFlags.NonPublic Or BindingFlags.InvokeMethod, Nothing, cgvp, New Object(){})
            End Using
        End Sub
    End Class
End Namespace
